cpp/src/quantiles/tdigest/tdigest_aggregation.cu

```diff
@@ -437,7 +519,7 @@ CUDF_KERNEL void generate_cluster_limits_kernel(int delta,
 
     // compute the weight we will be at in the input values just before closing off the current
     // cluster (because adding the next value will cross the current limit).
-    // NOTE: can't use structured bindings here.
+    // NOTE: can't use structured bindings here
     thrust::tie(nearest_w, nearest_w_index) = nearest_weight(next_limit, group_index);
```

Comment:
Optional: Might be worth using `cudf::std::tie` if the return type of `nearest_weight` is a `cuda::std::tuple` now instead of `thrust::tuple`